code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -10,7 +10,7 @@
 	. = ..()
 	recursive_organ_check(src)
 
-/obj/structure/closet/secure_closet/freezer/open(mob/living/user, force = TRUE)
+/obj/structure/closet/secure_closet/freezer/open(mob/living/user, force = FALSE)
```

Comment:
I believe there are a few others I goofed on, do you mind fixing those as well?

Reply:
sure lemme check